ui/components/nance/CompetitorPreview.tsx

```diff
@@ -0,0 +1,41 @@
+import { NFT, ThirdwebNftMedia } from '@thirdweb-dev/react'
+import { useEffect, useState } from 'react'
+
+type CompetitorPreviewProps = {
+  teamId: any
+  teamContract?: any
+}
+
+export function CompetitorPreview({
+  teamId,
+  teamContract,
+}: CompetitorPreviewProps) {
+  const [teamNFT, setTeamNFT] = useState<NFT>()
+
+  useEffect(() => {
+    async function getTeamNFT() {
+      const nft = await teamContract.erc721.get(teamId)
+      setTeamNFT(nft)
+    }
+
+    if (teamContract?.erc721?.get && teamId) {
+      getTeamNFT()
+    }
+  }, [teamId, teamContract])
+
+  return (
+    <div className="flex items-center gap-5">
+      {teamNFT && (
+        <div className="flex items-center">
+          <ThirdwebNftMedia
```

Comment:
I've ran into issues before with the <ThirdwebNftMedia/> component, if the image doesn't render then we'll need to switch to <MediaRenderer/>